src/include/miopen/solver.hpp

```diff
@@ -108,6 +108,8 @@ struct SolverBase
     /// run-time parameters.
     virtual bool IsDynamic() const { return false; }
 
+    static constexpr float wti_cant_be_computed = -2;
```

Comment:
```suggestion
    static constexpr float wti_unknown = -2.0f;
```
I would rather prefer a `#define`, it would be clear that this is a constant value, or leave it as is. This thing with constexpr just confuses everything.